config/localization/en/app.json

```diff
@@ -1526,31 +1526,20 @@
       "YOURE_IN": "You're in {LEAGUE}"
     },
     "VAULTS": {
-      "VAULT_DESCRIPTION": "Vaults provide automated liquidity on all dYdX markets. Vaults aim for a market-neutral position by quoting both sides of the book. P&L will vary based on market conditions and there's a risk of losing some or all of the USDC deposited.",
-      "LP_VAULT": "LP Vault",
       "VAULT": "Vault",
-      "ASSET_VAULT": "{ASSET} Vault",
-      "VAULTS": "Vaults",
-      "VAULT_DETAILS": "Vault Details",
-      "VAULT_FAQS": "Vault FAQs",
-      "VAULT_BALANCE": "Vault Balance",
-      "VAULT_POSITION": "Vault Position",
-      "VAULT_LEVERAGE": "Vault Leverage",
-      "VAULT_ALL_TIME_PNL": "All-time Vault P&L",
+      "YOUR_ALL_TIME_PNL": "Your All-time P&L",
```

Comment:
nit: `All-Time`?

Reply:
for some reason Dan has the t always lowercase in the designs.  I guess I'll leave it that way until someone complains